CEGameApp.API/Models/Users.cs

```diff
@@ -8,5 +10,21 @@ public class Users
 
         public int AccessLevel { get; set; }
 
+        public byte[] PasswordHash { get; set; }
```

Comment:
do we care that this is an array?  consider changing the type to IEnumerable<byte>, because who knows, you may need this to be a list or set at some point.

Reply:
going to leave it as bytes for now since this is only for demo purposes.